dtreeviz/models/sklearn_decision_trees.py

```diff
@@ -36,7 +37,7 @@ def get_features(self):
         return self.tree_model.tree_.feature
 
     def criterion(self):
-        return self.tree_model.criterion.upper()
+        return criterion_remapping.get(self.tree_model.criterion, self.tree_model.criterion)
```

Comment:
I wonder if we should just make a function that automatically gets the default value to hide the double reference to the criterion in the function call.

Reply:
Done! https://github.com/parrt/dtreeviz/pull/245/commits/c0f6a0fe6b620bd85a4a8a3f0fd61c68271e866b